tests/test_cases/test_discovery/test_discovery.py

```diff
@@ -89,6 +90,51 @@ async def access_signal(dut):
                          dut.stream_in_data.value.integer, 1))
 
 
+@cocotb.test(skip=cocotb.LANGUAGE in ["vhdl"])
+async def access_type_bit_verilog(dut):
+    """Access type bit in SystemVerilog"""
+    await Timer(1, "step")
+    assert dut.mybit.value == 1, "The default value was incorrect"
+    dut.mybit <= 0
+    await Timer(1, "ns")
+    assert dut.mybit.value == 0, "The assigned value was incorrect"
+
+    assert dut.mybits.value == 0b11, "The default value was incorrect"
+    dut.mybits <= 0b00
+    await Timer(1, "ns")
+    assert dut.mybits.value == 0b00, "The assigned value was incorrect"
+
+    assert dut.mybits_uninitialized.value == 0b00, "The default value was incorrect"
+    dut.mybits_uninitialized <= 0b11
+    await Timer(1, "ns")
+    assert dut.mybits_uninitialized.value == 0b11, "The assigned value was incorrect"
+
+
+@cocotb.test(skip=cocotb.LANGUAGE in ["vhdl"])
+async def access_type_bit_verilog_metavalues(dut):
+    """Access type bit in SystemVerilog with metavalues that the type does not support.
+
+    Note that some simulators (wrongly) allow metavalues even for bits when taking the VPI route.
+    The metavalues still may show up as `0` and `1` in HDL (Xcelium and Riviera).
+    """
+    await Timer(1, "ns")
+    dut.mybits <= BinaryValue("XZ")
+    await Timer(1, "ns")
+    print(dut.mybits.value.binstr)
+    if cocotb.SIM_NAME.lower().startswith(("icarus", "ncsim", "xmsim", "riviera")):
+        assert dut.mybits.value.binstr.lower() == "xz", "The assigned value was not as expected"
```

Comment:
Doesn't Xcelium have the same issue? I'm not sure what IUS and Xcelium's values would be.
```suggestion
    if cocotb.SIM_NAME.lower().startswith(("icarus", "ncsim", "xmsim")):
        assert dut.mybits.value.binstr.lower() == "xz", "The assigned value was not as expected"
    elif cocotb.SIM_NAME.lower().startswith(("riviera",)):
        assert dut.mybits.value.binstr.lower() == "10", "The assigned value was not as expected"
```

Reply:
IUS and Xcelium are what's behind `"ncsim", "xmsim"`.